doc/index.rst

```diff
@@ -7,14 +7,62 @@ Fast, Scalable and Easy Machine Learning With DAAL4PY
 Daal4py makes your Machine Learning algorithms in Python lightning fast and easy
 to use. It provides highly configurable Machine Learning kernels, some of which
 can be easily and efficiently scaled out to clusters of workstations.
-Internally it uses Intel(R) DAAL (Intel(R) Data Analytics Acceleration
-Library) to get best performance.
+Internally it uses Intel® DAAL (Intel® Data Analytics Acceleration
+Library) to deliver the  best performance.
 
-It's Easy
+Designed for Data Scientists and Framework Designers
 ---------
-A daal4py machine learning algorithm gets constructed with a rich set of
-parameters. Let's assume we want to find the initial set of centroids for kmeans.
-We create an algorithm and configure it for 10 clusters using the 'PlusPlus' method::
+daal4py was created to give data scientists the easist way to utilize Intel® DAAL's
+powerful machine learning building blocks directly in a high-productivity manner.
+A simplified API gives high-level abstractions to the user with minimal boilerplate,
+allowing for quick to write and easy to maintain code when utilizing Jupyter Notebooks.
+For scaling capabilities, daal4py also provides the ability to do distributed machine
+learning, giving a quick way to scale out without having to manually write the
```

Comment:
Don't compare it to anything in the DAAL world, e.g. that wiring is needed. Just say it's super-easy to scale out.